gulpfile.js

```diff
@@ -48,11 +50,15 @@ var COMPONENTS_DIR = BUILD_DIR + 'components/';
 var SINGLE_FILE_DIR = BUILD_DIR + 'singlefile/';
 var MINIFIED_DIR = BUILD_DIR + 'minified/';
 var FIREFOX_BUILD_DIR = BUILD_DIR + 'firefox/';
+var CHROME_BUILD_DIR = BUILD_DIR + '/chromium/';
```

Comment:
Remove the extra slash before `chromium` for consistency with the other variables above it.